### PR TITLE
[LoadTesting] Fix async client name

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

File: sdk/loadtesting/azure-developer-loadtesting/setup.py [hunk 1]
```diff
@@ -1,71 +1,79 @@
...
-from setuptools import setup, find_packages
-
+import os.path
+from io import open
+from setuptools import find_packages, setup
 
+# Change the PACKAGE_NAME only to change folder and different name
 PACKAGE_NAME = "azure-developer-loadtesting"
-PACKAGE_PPRINT_NAME = "Azure Developer LoadTesting"
+PACKAGE_PPRINT_NAME = "MyService Management"
```
Comment (prativen) [inline sdk/loadtesting/azure-developer-loadtesting/setup.py:16]: this does not look right

File: sdk/loadtesting/azure-developer-loadtesting/setup.py [hunk 2]
```diff
@@ -1,71 +1,79 @@
...
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: 3.11",
-        "License :: OSI Approved :: MIT License",
+        'Development Status :: 4 - Beta',
```
Comment (prativen) [inline sdk/loadtesting/azure-developer-loadtesting/setup.py:50]: This is not Beta?

File: sdk/loadtesting/azure-developer-loadtesting/README.md [hunk 3]
```diff
@@ -1,258 +1,63 @@
...
-from azure.identity import DefaultAzureCredential
-from azure.core.exceptions import HttpResponseError
+## Examples
 
-TEST_ID = "some-test-id"  
-TEST_RUN_ID = "some-testrun-id" 
-DISPLAY_NAME = "my-load-test-run"  
 
-client = LoadTestRunClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+Code samples for this package can be found at [MyService Management](https://docs.microsoft.com/samples/browse/?languages=python&term=Getting%20started%20-%20Managing&terms=Getting%20started%20-%20Managing) on docs.microsoft.com and [Samples Repo](https://aka.ms/azsdk/python/mgmt/samples)
```
Comment (prativen) [inline sdk/loadtesting/azure-developer-loadtesting/README.md:49]: This needs to be updated?

File: sdk/loadtesting/azure-developer-loadtesting/setup.py [hunk 4]
```diff
@@ -1,71 +1,79 @@
...
-    url="https://github.com/Azure/azure-sdk-for-python/tree/main/sdk",
-    keywords="azure, azure sdk",
+    description='Microsoft Azure {} Client Library for Python'.format(PACKAGE_PPRINT_NAME),
+    long_description=readme + '\n\n' + changelog,
+    long_description_content_type='text/markdown',
+    license='MIT License',
+    author='Microsoft Corporation',
+    author_email='azpysdkhelp@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python',
+    keywords="azure, azure sdk",  # update with search keywords relevant to the azure service / product
```
Comment (prativen) [inline sdk/loadtesting/azure-developer-loadtesting/setup.py:48]: keywords need to be updated?

File: sdk/loadtesting/azure-developer-loadtesting/setup.py (hunk 1 above)
Comment (annatisch) [reply sdk/loadtesting/azure-developer-loadtesting/setup.py:16]: Sorry - this isn't my change - the bot has added an auto-commit to the PR. I will revert this.

Comment (annatisch) [reply sdk/loadtesting/azure-developer-loadtesting/setup.py]: I've reverted the auto-commit now. Should be good to go.

**Review by msyyc — APPROVED** [review]

**Pull request merged**